search/include/pcl/search/flann_search.h

```diff
@@ -348,22 +348,22 @@ namespace pcl
 
         /** Epsilon for approximate NN search.
           */
-        float eps_;
+        float eps_{0.0f};
```

Comment:
You did not make any corresponding changes in flann_search.hpp?

Reply:
Weird omission by `clang-tidy`